aexpect/rss_client.py

```diff
@@ -234,7 +246,7 @@ def _receive_file_chunks(self, filename, timeout=60):
         finally:
             f.close()
 
-    def _send_msg(self, msg, timeout=60):
+    def _send_msg(self, msg, _timeout=60):
```

Comment:
Again, please delete the `timeout` or use `pylint: disable=`. Also in this case it'd probably make sense to implement the timeout :wink:

Reply:
Hehe sure, I am just sticking to the minimum modification policy right now. Even modifications of the smallest size usually require separate commits and are not part of the pure migration process + risk breaking more things for more people and require much more testing. I have made the argument usable for the next push.